docs/data/material/experimental-api/css-variables/css-variables.md

```diff
@@ -1,4 +1,4 @@
-# CSS variables
+# TODO merge with other pages
```

Comment:
@oliviertassinari I see this in https://mui.com/material-ui/experimental-api/css-variables/. I think you intended to add a comment here. As for the fix, should we remove this page or keep the heading as `CSS variables`?

Reply:
I think that the next step is to merge the two pages. For example https://mui.com/material-ui/experimental-api/css-variables/ has demos while there is not a single demo on the new pages. I would have expected many simple demos. So developers can play with it and understand what it does.